dialogue-annotations-example/src/main/java/com/palantir/myservice/example/MyService.java

```diff
@@ -76,13 +76,15 @@ void params(
             @Request.QueryParam("q3") Optional<String> query3,
             // Alias types are supported for @QueryParam and @Header
             @Request.QueryParam("q4") MyAliasType query4,
+            @Request.QueryParam("q5") List<MyAliasType> query5,
```

Comment:
We should add tests here that produce the generated code for this case.

https://github.com/palantir/dialogue/blob/bfa0a4b5bd70beb12c18004b2f3fb0a89bbf82a8/dialogue-annotations-processor/src/test/java/com/palantir/myservice/service/MyService.java

Reply:
I updated it